docker/resources/container-entrypoint-with-env

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 
+stop_container() {
+    # This script is running as PID=1. Stopping this
+    # process will stop the entire container.
+    exit 0
```

Comment:
I now realize that it is probably better to stop the Inmanta server gracefully instead of doing a hard kill. I will add this small change.